### ISSUE TITLE
Ops in some ONNX models

### ISSUE THREAD

**Issue by tungld** [clang-ykt/ONNF#46]
Here, we enumerate ops used in some popular models, and summarize their implementing status in ONNF. An op is available once its backend tests have passed.

(PR: means still in pull request, Y: Yes)

| Operator           | Available in ONNF | [MNIST]( https://s3.us.cloud-object-storage.appdomain.cloud/dlcompiler/model/mnist.onnx) | [ResNet50.BS1]( https://s3-api.us-geo.objectstorage.softlayer.net/dlcompiler/model/resnet50_bs1.onnx) |
|--------------------|-----------|-------|--------------|
| Add                | Y         | Y     |              |
| AveragePool        |           |       | Y            |
| BatchNormalization |           |       | Y            |
| Conv               |           | Y     | Y            |
| Gemm               | Y(PR)     |       | Y            |
| Identity           |           | Y     |              |
| MaxPool            |           | Y     | Y            |
| MatMul             |           | Y     |              |
| Relu               | Y         | Y     | Y            |
| Reshape            | Y         | Y     | Y            |
| Softmax            | Y         |       | Y            |
| Sum                | Y         |       | Y            |
| Transpose          |           | Y     |              |
| Unsqueeze          |           | Y     |              |
|                    |           |       |              |

**Comment by tungld** [comment]
FYI, @AlexandreEichenberger, @doru1004, @chentong319.

**Comment by bhack** [comment]
We are discussing OPS in MLIR. If you are interested please join in the thread at https://llvm.discourse.group/t/development-of-high-level-tensor-compute-primitives-dialect-s-and-transformations/388
